src/components/RemsInterface/RemsInterface.jsx

```diff
@@ -27,9 +27,11 @@ export default class RemsInterface extends Component {
     return options;
   }
 
-  sendMessage(destination) {
-      console.log(this.props.specialtyRxBundle);
-    axios.post(destination, this.props.specialtyRxBundle, this.getAxiosOptions());
+  async sendRemsMessage() {
```

Comment:
I'm personally not a fan of async/await since it pauses execution.  But it's not super important that this section of the app doesn't get held up in any way.